targets/TARGET_STM/TARGET_STM32L0/device/stm32l0xx_hal_conf.h

```diff
@@ -132,7 +132,7 @@
   * @brief Internal Low Speed oscillator (LSI) value.
   */
 #if !defined  (LSI_VALUE)
- #define LSI_VALUE  (37000U)       /*!< LSI Typical Value in Hz*/
+ #define LSI_VALUE  (38000U)       /*!< LSI Typical Value in Hz*/
```

Comment:
Reference Manual says: The clock frequency is around 37 kHz
Data Sheet adds details about the value: min 26 / Typ 38 / Max 56
So you could change value from 37 to 38, this will not change anything... Some boards will be at 26, other at 56...

Reply:
You are right and on some diagrams it's marked as 40kHz. Won't change anything, but it will be consistent to data sheet.